projects/kit/components/items-with-more/items-with-more.directive.ts

```diff
@@ -1,24 +1,45 @@
 import type {OnChanges} from '@angular/core';
 import {Directive, Input} from '@angular/core';
+import {tuiInjectElement} from '@taiga-ui/cdk/utils/dom';
 import {Subject} from 'rxjs';
 
 @Directive({
     standalone: true,
+    host: {
+        '[class._multiline]': 'linesLimit > 1',
+        '[style.--t-min-width.px]': 'maxWidth()',
+    },
 })
 export class TuiItemsWithMoreDirective implements OnChanges {
+    private readonly el = tuiInjectElement();
+    private moreSide: 'end' | 'start' = 'end';
+
     @Input()
     public itemsLimit = Infinity;
 
     @Input()
     public required = -1;
 
     @Input()
-    public side: 'end' | 'start' = 'end';
+    public linesLimit = 1;
 
     // TODO: refactor to signal inputs after Angular update
     public readonly change$ = new Subject<void>();
 
+    @Input('side')
+    public set setSide(side: 'end' | 'start') {
+        this.moreSide = side;
```

Comment:
This is messy, I think we only need it in one place in the service, let's just check if we have multiline there?